### PR TITLE
Added Opentracing dependencies

### PR DESCRIPTION
**Pull request by prydin**
Added missing Opentracing dependencies. This fixes the issue with traces not showing up.

### CODE REVIEW

File: wavefront-spring-boot/pom.xml [hunk 1]
```diff
@@ -40,6 +40,18 @@
       <optional>true</optional>
     </dependency>
 
+
+    <dependency>
```
Comment (snicoll) [inline wavefront-spring-boot/pom.xml:44]: Those are runtime dependencies so they should not be defined here but rather on the starter itself, or, by the application as we ultimately want some flexibility and opt-in for tracing.

File: wavefront-spring-boot/pom.xml [hunk 2]
```diff
@@ -40,6 +40,18 @@
...
+      <groupId>io.opentracing.contrib</groupId>
+      <artifactId>opentracing-spring-cloud-starter</artifactId>
+      <version>0.3.12</version>
```
Comment (snicoll) [inline wavefront-spring-boot/pom.xml:47]: Versions are not defined here but in the parent.

File: wavefront-spring-boot/pom.xml [hunk 3]
```diff
@@ -40,6 +40,18 @@
...
+    </dependency>
+    <dependency>
+      <groupId>io.opentracing</groupId>
+      <artifactId>opentracing-api</artifactId>
```
Comment (snicoll) [inline wavefront-spring-boot/pom.xml:51]: Isn't that provided by the `opentracing-spring-cloud-starter` already?

**Comment by panghy** [comment]
Me and @snicoll chatted and I think ultimately it would be best if it's based on classpath detection between OT and Sleuth. It does mean someone would have to pick (so more lines but then again there's initialzr).

**Comment by snicoll** [comment]
I've [clarified the setup](https://github.com/wavefrontHQ/wavefront-spring-boot/commit/f8a845249b6228328e9315f8059a57b858d6272f) for tracing with OpenTracing and added the starter to the sample so that it is more obvious that an additional dependency is required. This should effectively supersede this considering we are working on supporting Sleuth as well.

Thanks for the PR, in any case!

**Pull request closed without merging**